env.default

```diff
@@ -50,7 +50,7 @@ BASKET_URL=https://basket-dev.allizom.org
 # CSP config
 CSP_CHILD_SRC=" 'self' https://www.youtube.com https://www.youtube-nocookie.com "
 CSP_CONNECT_SRC=" * "
-CSP_DEFAULT_SRC=" 'none' "
+CSP_FRAME_ANCESTORS=" 'self' "
```

Comment:
Just noticed this is still not added as default, so fixed it whilst here - related to https://github.com/MozillaFoundation/foundation.mozilla.org/issues/10768

Reply:
Should we be removing `CSP_DEFAULT_SRC=" 'none' "` to add this?

Reply:
:facepalm: wrong line, my bad

Reply:
ea005e40e217a81e5186210f11046d2d78a760fe